src/somef/process_repository.py

```diff
@@ -20,14 +33,14 @@ def rate_limit_get(*args, backoff_rate=2, initial_backoff=1, **kwargs):
     date = ""
     while rate_limited:
         response = requests.get(*args, **kwargs)
-        data = response
-        date = data.headers["date"]
-        rate_limit_remaining = data.headers["x-ratelimit-remaining"]
-        epochtime = int(data.headers["x-ratelimit-reset"])
-        date_reset = datetime.fromtimestamp(epochtime)
-        logging.info("Remaining GitHub API requests: " + rate_limit_remaining + " ### Next rate limit reset at: " + str(
+        date = response.headers["Date"]
```

Comment:
Change of capitaliation won't arise problems? (`Date` vs `date`)

Reply:
Yes, it might well be. The [documentation of the GitHub REST Resources](https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28) states the headers should be in small caps, but I'm receiving them differently. I'll be sure to check this more carefully

Reply:
ok! Usually non-cappital letters make sense to me, but I just want to make sure we don't add breaking changes